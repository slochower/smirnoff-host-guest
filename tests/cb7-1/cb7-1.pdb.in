
    parm ./tests/cb7-1/cb7-1.prmtop
    trajin ./tests/cb7-1/cb7-1.rst7
    trajout ./tests/cb7-1/cb7-1.pdb conect
    