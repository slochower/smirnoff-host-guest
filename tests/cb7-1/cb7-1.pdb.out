
CPPTRAJ: Trajectory Analysis. V17.00
    ___  ___  ___  ___
     | \/ | \/ | \/ | 
    _|_/\_|_/\_|_/\_|_

| Date/time: 02/28/18 20:46:59
| Available memory: 1.174 GB

INPUT: Reading input from './tests/cb7-1/cb7-1.pdb.in'
  [parm ./tests/cb7-1/cb7-1.prmtop]
	Reading './tests/cb7-1/cb7-1.prmtop' as Amber Topology
	Radius Set: modified Bondi radii (mbondi)
  [trajin ./tests/cb7-1/cb7-1.rst7]
	Reading './tests/cb7-1/cb7-1.rst7' as Amber Restart
  [trajout ./tests/cb7-1/cb7-1.pdb conect]
	Writing './tests/cb7-1/cb7-1.pdb' as PDB
---------- RUN BEGIN -------------------------------------------------

PARAMETER FILES (1 total):
 0: cb7-1.prmtop, 4662 atoms, 1503 res, box: Orthogonal, 1503 mol, 1500 solvent

INPUT TRAJECTORIES (1 total):
 0: 'cb7-1.rst7' is an AMBER restart file with velocity info, Parm cb7-1.prmtop (Orthogonal box) (reading 1 of 1)
  Coordinate processing will occur on 1 frames.

OUTPUT TRAJECTORIES (1 total):
  'cb7-1.pdb' (1 frames) is a PDB file with CONECT records

BEGIN TRAJECTORY PROCESSING:
Warning: No PDB space group specified.
----- cb7-1.rst7 (1-1, 1) -----
100% Complete.

Read 1 frames and processed 1 frames.
TIME: Avg. throughput= 74.6213 frames / second.

ACTION OUTPUT:

RUN TIMING:
TIME:		Init               : 0.0000 s (  0.06%)
TIME:		Trajectory Process : 0.0134 s ( 78.96%)
TIME:		Action Post        : 0.0000 s (  0.00%)
TIME:		Analysis           : 0.0000 s (  0.00%)
TIME:		Data File Write    : 0.0000 s (  0.01%)
TIME:		Other              : 0.0036 s (  0.21%)
TIME:	Run Total 0.0170 s
---------- RUN END ---------------------------------------------------
TIME: Total execution time: 0.0281 seconds.
--------------------------------------------------------------------------------
To cite CPPTRAJ use:
Daniel R. Roe and Thomas E. Cheatham, III, "PTRAJ and CPPTRAJ: Software for
  Processing and Analysis of Molecular Dynamics Trajectory Data". J. Chem.
  Theory Comput., 2013, 9 (7), pp 3084-3095.

